meshnets/main.py

```diff
@@ -9,32 +9,37 @@
 
 from meshnets.utils import data_loading
 from meshnets.utils import data_processing
+#from meshnets.utils import data_visualization
 
 FLAGS = flags.FLAGS
-
-flags.DEFINE_string('input_object', 'data/motorBike.obj',
+flags.DEFINE_string('input_object', 'data/blob.vtk',
```

Comment:
Remove the hardcoded path. Two options, either you use `os.path.join` or we can set this to `None` and mark the flag as required.